### DISCUSSION TITLE
Problems to install it

### DISCUSSION THREAD

**Discussion by Miquel-code** [TheLastBen/fast-stable-diffusion#1496] [General]
I have been used the **1.5 version** but now I have this problem when I try to use Stable Diffusion:

![image](https://user-images.githubusercontent.com/110945646/215011060-350b1473-683f-477f-bc29-ea7240cca9cc.png)

**I delete and reload "_SD_"** folder many times but that **doesn't fix the problem**. Please **help me!**

Maybe the problem is the missing file **_"blocks.py.bat"_**.

If that's the problem, how can I fix the problem?

**Comment by TheLastBen** [comment]
You are using an old colab, always use the new one : https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb

**Reply by Miquel-code** [reply]
OMG **The big boss** has answer my question. **I appreciate your help** a lot, now I can use Stable Diffussion again.

But, how can I know when the "_Notebook_" has changed?
How can I download the new versions and use them on Google Clab?

![image](https://user-images.githubusercontent.com/110945646/215357364-9bbd37ac-33e8-44ec-8cc6-dc445bcf8e4c.png)
